releasenotes/source/index.rst

```diff
@@ -2,9 +2,59 @@
 terraform-provider-opentelekomcloud Release Notes
 =================================================
 
-.. toctree::
-   :maxdepth: 1
+.. release-notes::
 
-   current
-   older
+1.24.2
+------
 
+New Features
```

Comment:
do you really want to have it all in the index.rst? Overall idea is that those files know nothing about changes at all. I suggest to drop them completely and i.e. set earliest_version to 1.24.3

Reply:
done